### PR TITLE
[doc]Supplement the 'X-API-KEY' parameter for each interface that operates admin.

### PR DESCRIPTION
**Pull request by Miss-you**
Supplement the 'X-API-KEY' parameter for each interface that operates admin.

NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: doc/admin-api-cn.md [hunk 1]
```diff
@@ -19,11 +19,13 @@
 
 # 目录
 
-* [Route](#route)
-* [Service](#service)
-* [Consumer](#consumer)
-* [Upstream](#upstream)
-* [SSL](#ssl)
+- [目录](#%e7%9b%ae%e5%bd%95)
```
Comment (membphis) [inline doc/admin-api-cn.md:22]: we'd better use Chinese here. `#%e7%9b%ae%e5%bd%95` is not a good choice.

Comment (Miss-you) [reply doc/admin-api-cn.md:22]: Hi, after submitting last night I noticed that due to the vs code markdown plugin, it automatically generates a list of headings that overwrite the past ones, I need to fix that part!

**Comment by Miss-you** [comment]
Hi, after submitting last night I noticed that due to the vs code markdown plugin, it automatically generates a list of headings that overwrite the past ones, I need to fix that part!

**Comment by membphis** [comment]
@Miss-you many thx

**Pull request merged**